app/components/Onboarding/Onboarding.js

```diff
@@ -129,11 +172,13 @@ const Onboarding = ({
         return (
           <FormContainer
             title='Encrypt your seed'
-            description='Totally optional, but we encourage it. Set a password that will be used to encrypt your wallet seed' // eslint-disable-line
+            description="Totally optional, but we encourage it. Set a password that will be used to encrypt your wallet seed" // eslint-disable-line
             back={() => changeStep(6)}
             next={() => {
               // dont allow the user to move on if the confirmation password doesnt match the original password
-              if (newAezeedPasswordProps.showAezeedPasswordConfirmationError) { return }
+              if (newAezeedPasswordProps.showAezeedPasswordConfirmationError) {
```

Comment:
are you breaking these conditionals into multiple lines for any particular reason?